pkgs/servers/nosql/cassandra/generic.nix

```diff
@@ -98,11 +102,18 @@ stdenv.mkDerivation rec {
       fi
     done
 
-    wrapProgram $out/bin/cqlsh --prefix PATH : ${python}/bin
-
     runHook postInstall
   '';
 
+  postFixup = ''
+    # Remove cassandra bash script wrapper.
+    # The wrapper searches for a suitable python version and is not necessary with Nix.
+    rm $out/bin/cqlsh
+    # Make "cqlsh.py" accessible by invoking "cqlsh"
+    ln -s $out/bin/cqlsh.py $out/bin/cqlsh
+    wrapPythonPrograms
```

Comment:
Could you show me (`cat`) `$out/bin/cqlsh` and `$out/bin/.cqlsh_wrapped` (just `import` part)?

Reply:
Sure, here:

```
> cat result/bin/cqlsh

#! /nix/store/4bj2kxdm1462fzcc2i2s4dn33g2angcc-bash-5.2p32/bin/bash -e
PATH=${PATH:+':'$PATH':'}
PATH=${PATH/':''/nix/store/4c6i7flrfki2816qaw5zplhya7s1j47j-python3.12-geomet-1.1.0/bin'':'/':'}
PATH='/nix/store/4c6i7flrfki2816qaw5zplhya7s1j47j-python3.12-geomet-1.1.0/bin'$PATH
PATH=${PATH#':'}
PATH=${PATH%':'}
export PATH
PATH=${PATH:+':'$PATH':'}
PATH=${PATH/':''/nix/store/69mqx2f5wxk8yrks3gm6klhh7c0xgvkj-cassandra-4.1.2/bin'':'/':'}
PATH='/nix/store/69mqx2f5wxk8yrks3gm6klhh7c0xgvkj-cassandra-4.1.2/bin'$PATH
PATH=${PATH#':'}
PATH=${PATH%':'}
export PATH
PATH=${PATH:+':'$PATH':'}
PATH=${PATH/':''/nix/store/04gg5w1s662l329a8kh9xcwyp0k64v5a-python3-3.12.4/bin'':'/':'}
PATH='/nix/store/04gg5w1s662l329a8kh9xcwyp0k64v5a-python3-3.12.4/bin'$PATH
PATH=${PATH#':'}
PATH=${PATH%':'}
export PATH
export PYTHONNOUSERSITE='true'
exec -a "$0" "/nix/store/69mqx2f5wxk8yrks3gm6klhh7c0xgvkj-cassandra-4.1.2/bin/.cqlsh.py-wrapped"  "$@" 
> head -20 result/bin/.cqlsh.py-wrapped
#!/nix/store/04gg5w1s662l329a8kh9xcwyp0k64v5a-python3-3.12.4/bin/python3

# Licensed to the Apache Software Foundation (ASF) under one
# or more contributor license agreements.  See the NOTICE file
# distributed with this work for additional information
# regarding copyright ownership.  The ASF licenses this file
# to you under the Apache License, Version 2.0 (the
# "License"); you may not use this file except in compliance
# with the License.  You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

import sys;import site;import functools;sys.argv[0] = '/nix/store/69mqx2f5wxk8yrks3gm6klhh7c0xgvkj-cassandra-4.1.2/bin/cqlsh.py';functools.reduce(lambda k, p: site.addsitedir(p, k), ['/nix/store/q74p7y0s12nz1x4a5q0by0dvkjfd3vdx-python3.12-cassandra-driver-3.29.1/lib/python3.12/site-packages','/nix/store/d8kyggpkgp2k28sc8x1h8alwlpwlqaf2-python3.12-six-1.16.0/lib/python3.12/site-packages','/nix/store/4c6i7flrfki2816qaw5zplhya7s1j47j-python3.12-geomet-1.1.0/lib/python3.12/site-packages','/nix/store/3k6fxb2pr96xdv5nbhcd50hfzhg4vq2b-python3.12-click-8.1.7/lib/python3.12/site-packages'], site._init_pathinfo());
import cmd
```

Thanks for taking time to review this.